### ISSUE TITLE
Adding status labels or icon to event participants

### ISSUE THREAD

**Issue by AlexGeniusMan** [RTUITLab/ITLab#12]
At the moment getting status is not obvious
![image](https://user-images.githubusercontent.com/47246896/132942045-4f0c880e-37fd-4d87-b081-8e6b20d8b40f.png)

I suggest adding status label ('Подтвержденный', 'В ожидании', etc) or status icon next to the member's role label

**Comment by IvLaptev** [comment]
@AlexGeniusMan, по идее все 3 статуса сейчас выделяются цветом.

1. Подавая заявку, человек видит, что та серая. После подтверждения она станет зелёной.
2. А те, кто приглашает на события, вполне знают, что жёлтый цвет значит, что человек приглашён.

Думаешь цветовой индикации недостаточно?

**Comment by CAPCHIK** [comment]
Как по мне - в целом хватит и текущего, ибо привыкнуть к этому - пара раз наткнуться

**Comment by AlexGeniusMan** [comment]
> @AlexGeniusMan Учитывая, что репы наши часто открытые - не стоит выкладывать скриншоты работы внутренних сервисов, в которых есть данные о людях. В данном случае на скрине можно затереть фио + почту, или на момент скрина в F12 поменять ФИО на что-то по типу `Крокодилов Гена Гармошкович`

Да, это понятно. В данном случае я специально не стал замазывать наши с Кириллом почты, так как они в открытом доступе висят. Но спасибо, на будущее полезно

**Comment by AlexGeniusMan** [comment]
> @AlexGeniusMan, по идее все 3 статуса сейчас выделяются цветом.
> 
> 1. Подавая заявку, человек видит, что та серая. После подтверждения она станет зелёной.
> 2. А те, кто приглашает на события, вполне знают, что жёлтый цвет значит, что человек приглашён.
> 
> Думаешь цветовой индикации недостаточно?

Я говорю лишь об очевидности. Для меня, например, не было очевидным то, что нужно навести курсор на бейдж, поэтому пришлось дергать Максима, чтобы узнать статус

**Comment by AlexGeniusMan** [comment]
> Как по мне - в целом хватит и текущего, ибо привыкнуть к этому - пара раз наткнуться

Опять же, вам решать :)
